stubs/six/@python2/six/moves/__init__.pyi

```diff
@@ -40,6 +39,7 @@ from . import (
     _dummy_thread,
     _thread,
     configparser,
+    copyreg,
```

Comment:
```suggestion
    copyreg as copyreg,
```

I'm not sure why the other imports are not re-exported. The Python import mechanism is quite arcance. Anyway, better to make it explicit.

Reply:
done 😄